source/connect/connection-options/connection-pools.txt

```diff
@@ -103,39 +208,67 @@ A request that waits more than the length of time defined by
+the following actions:
+
+- Closes all idle sockets in the connection pool
+- Closes all sockets that are in use as they are returned to the pool
+- Closes all sockets that are in use only when the associated operations
+  complete
+
+Calling ``MongoClient.close()`` closes only inactive sockets and does not
+directly terminate any ongoing operations. 
+
+.. note::
+
+   The ``MongoClient.close()`` method does close existing sessions and
+   transactions, which might indirectly affect the behavior of ongoing
+   operations and open cursors.
 
 Avoid Socket Timeouts
 ---------------------
 
 Having a large connection pool does not always reduce reconnection
-requests. Consider the following example:
+requests. Consider the following example scenario:
 
-An application has a connection pool size of 5 sockets and has the
-``socketTimeoutMS`` option set to 5000 milliseconds. Operations occur,
-on average, every 3000 milliseconds, and reconnection requests are
-frequent. Each socket times out after 5000 milliseconds, which means
-that all sockets must do something during those 5000 milliseconds to
-avoid closing.
+- An application has a connection pool size of 5 sockets and has the
+  ``socketTimeoutMS`` option set to 5000 milliseconds. 
+- Operations occur, on average, every 3000 milliseconds, and reconnection
+  requests are frequent. 
+- Each socket times out after 5000 milliseconds, which means that all sockets
+  must do something during those 5000 milliseconds to avoid closing.
 
-One message every 3000 milliseconds is not enough to keep the sockets
-active, so several of the sockets will time out after 5000 milliseconds.
-To avoid excessive socket timeouts, reduce the number of connections
-that the driver can maintain in the connection pool by specifying the
-``maxPoolSize`` option.
+In this scenario, each socket times out after 5000 milliseconds, requiring
+activity within this timeout period to avoid closure. However, one message every
+3000 milliseconds isn't enough to keep all sockets active, causing several of
+them to time out.
 
-To specify the optional ``maxPoolSize`` setting for your ``MongoClient``, declare
-it in the ``options`` object of the constructor as follows:
+To avoid excessive socket timeouts, reduce the number of connections that the
+driver can maintain in the connection pool by specifying the ``maxPoolSize``
+option. To learn how to set this option, see the
+:ref:`node-connection-pool-max-pool-size` section.
 
-.. code-block:: javascript
+API Documentation
+-----------------
 
-   const client = new MongoClient(uri, {
-     maxPoolSize: <integer value>,
-   });
\ No newline at end of file
+For more information about creating a ``MongoClient`` object with the 
+{+driver-short+} and specifying options, see the following API documentation: 
+
+- `MongoClient <{+api+}/classes/MongoClient.html>`__
+- `MongoClientOptions <{+api+}/interfaces/MongoClientOptions.html>`__
```

Comment:
S: Since the Options page is so long, it would be nice to include the specific links to each of the relevant options.